### PR TITLE
Second version of my portfolio

### PR DESCRIPTION
**Pull request by Yothu**
his is the first version of my third project, PERSONAL PORTFOLIO.

Files previously added:

- .gitignore
- MIT.md
- README.md
- app_screenshot.png
- index.html
- css/style.css
- linters for HTML/CSS
- main background image
- social media logos

Files updated since last master version:

- index.html
- css/style.css

Files created since last master version:

- image/Indicator.png
- image/framework.png
- image/language.png
- image/skill.png
- image/background/about-me-background.png

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -156,3 +175,313 @@
...
+.proj {
+  background-color: lightskyblue;
+}
+
+.proj-title {
+  background-color: mediumblue;
+}
+
+  ------------------------------------ */
+
```
Comment (andersonlebon) [inline css/style.css]: - [x] To make your code clear and Kindly remove all commented code.

File: css/style.css [hunk 2]
```diff
@@ -156,3 +175,313 @@
...
+}
+
+.abi-data {
+  background-color: chartreuse;
+}
+
+.abi-data h3 {
+  background-color: cornflowerblue;
+}
```
Comment (andersonlebon) [inline css/style.css]: - [x] To make your code clear and Kindly remove all commented code.

File: css/style.css [hunk 3]
```diff
@@ -156,3 +175,313 @@
...
+  display: grid;
+  grid-template-columns: calc(100% - 48px);
+  justify-content: space-evenly;
+}
+
+.get-touch {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
```
Comment (andersonlebon) [inline css/style.css]: - [x] To make your code clear and Kindly remove all commented code.

File: index.html [hunk 4]
```diff
@@ -55,5 +55,187 @@ <h2 class="profession">
...
+        <li class="medium-logo">
+          <a href="https://medium.com/@dhaivy"><img src="./images/social/medium.svg" alt="medium"></a>
+        </li>
+        <li class="github-logo">
+          <a href="https://github.com/Yothu"><img src="./images/social/github.svg" alt="github"></a>
+        </li>
+        <li class="angelist-logo">
+          <a href="https://angel.co/u/david-almontes"><img src="./images/social/angellist.svg" alt="angelist"></a>
+        </li>
+      </ul>
```
Comment (andersonlebon) [inline index.html]: - [x] To make your code clear and Kindly remove all commented code.

File: index.html [hunk 5]
```diff
@@ -55,5 +55,187 @@ <h2 class="profession">
...
+              <li>Bootstrap</li>
+              <li>Ruby on rails</li>
+              <li>RSpec</li>
+              <li>Capybara</li>
+              <li>Selenium</li>
+            </ul>
+          </div>
+        </li>
+        <li class="ability-box">
+          <img src="./images/skill.png" alt="skills-icon">
```
Comment (andersonlebon) [inline index.html]: - [x] One of the requirements of the project says that we need to Put the about section in a `<section>` tag. Kindly I would suggest removing all the `about me` elated code from the `<nav>` tag and putting it in a `<section>` container. Remove also the `about section` in `my work section`. you will end up with this format
``` 
<section class="work">
             ......content
 </section>
 <section class="about-me">
             ......content
 </section>

```

File: css/style.css [hunk 6]
```diff
@@ -156,3 +175,313 @@
...
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  display: flex;
+  flex-wrap: wrap;
+  column-gap: 8px;
+  row-gap: 8px;
+  justify-content: space-evenly;
```
Comment (andersonlebon) [inline css/style.css]: - [x] Compare your design on the right side and the template one on the left side then:
- Reduce the space between technologies(ruby on rails, bootstrap, HTML, CSS...) by changing the `justify-content: space-evenly` with `justify-content: space-center`

Screenshot from 2021-11-26 03-57-17

**Review by andersonlebon — CHANGES_REQUESTED** [review]
Hi @Yothu 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Good use of `flex` and `grid` properties :heavy_check_mark: 
- Your design looks perfect :100: :+1:  :heavy_check_mark: 
## Required Changes ♻️

- [x] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elyor-doniyorov — APPROVED** [review]
Hi @Yothu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**